config/projects-prod/ctf-swnts-project.yaml

```
@@ -36,11 +35,12 @@ parameters:
   # DO NOT EDIT ANYTHING BELOW THIS LINE ------------------------------------------------------------------------
 
   AccountAdminArns:
-    - '{{stack_group_config.sso_admin_role.arn}}'
+    - "{{stack_group_config.sso_admin_role.arn}}"
     - !stack_output_external workflows-nextflow-ci-service-account::ServiceRoleArn
-  TemplateRootUrl: 'https://{{stack_group_config.admincentral_cf_bucket}}.s3.amazonaws.com'
+  TemplateRootUrl: "https://{{stack_group_config.admincentral_cf_bucket}}.s3.amazonaws.com"
   TowerForgePolicyArn: !stack_output_external nextflow-forge-iam-policy::NextFlowForgePolicyArn
   TowerLaunchPolicyArn: !stack_output_external nextflow-launch-iam-policy::NextFlowLaunchPolicyArn
+  EnableScratchDataExpiration: Disabled
```

@BrunoGrandePhD Is this what needs to happen for Sasha?
```suggestion
  EnableScratchDataExpiration: Enabled
```